docker-compose-no-volumes.yml

```diff
@@ -0,0 +1,31 @@
+# This docker compose file is used to create a backend environment without volumes.
+version: "3.7"
+
+services:
+  backend:
+    image: openmrs/openmrs-reference-application-3-backend:${TAG:-nightly}
+    depends_on:
+      - db
+    environment:
+      OMRS_CONFIG_MODULE_WEB_ADMIN: "true"
+      OMRS_CONFIG_AUTO_UPDATE_DATABASE: "true"
+      OMRS_CONFIG_CREATE_TABLES: "true"
+      OMRS_CONFIG_CONNECTION_SERVER: db
+      OMRS_CONFIG_CONNECTION_DATABASE: openmrs
+      OMRS_CONFIG_CONNECTION_USERNAME: ${OPENMRS_DB_USER:-openmrs}
+      OMRS_CONFIG_CONNECTION_PASSWORD: ${OPENMRS_DB_PASSWORD:-openmrs}
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/openmrs"]
+      timeout: 5s
+    ports:
+      - 9000:8080
```

Comment:
Is there a reason not to use 8080 as a default here?

Reply:
We used 8080 port to run the frontend (with `yarn start --sources 'packages/esm-*-app/` ). So the local backend was run on a different port.

Reply:
Right... I mean, for this use case, we don't even need an exposed port do we?

Reply:
But to check whether the initial data generation is completed, we need an exposed port. Isn't it? 

 (In order to use something similar to `$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/openmrs/login.htm)" != "200"`)

Reply:
I changed the exposed port number to 8080 with [f572b1a]